src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -7700,4 +7700,10 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="WRN_InlineArrayConversionOperatorNotUsed_Title" xml:space="preserve">
     <value>Inline array conversion operator will not be used for conversion from expression of the declaring type.</value>
   </data>
+  <data name="WRN_InlineArrayNotSupportedByLanguage" xml:space="preserve">
+    <value>'Inline arrays' language feature is not supported for inline array types with element field which is either a 'ref' field, or has type that is not valid as a type argument.</value>
```

Comment:
Why a separate error for `ref` fields? Couldn't `ERR_InlineArrayUnsupportedElementFieldModifier` be used for that?

Reply:
> Why a separate error for ref fields? Couldn't ```ERR_InlineArrayUnsupportedElementFieldModifier``` be used for that?

It cannot. The ```ERR_InlineArrayUnsupportedElementFieldModifier``` is an error. This is a warning. And it is reported for scenarios other than `ref` fields as well.